core/src/main/java/io/grpc/Metadata.java

```diff
@@ -204,9 +203,9 @@ public boolean isSerializable() {
     Preconditions.checkState(serializable, "Can't serialize raw metadata");
     byte[][] serialized = new byte[store.size() * 2][];
     int i = 0;
-    for (Map.Entry<String, MetadataEntry> entry : store.entries()) {
-      serialized[i++] = entry.getValue().key.asciiName();
-      serialized[i++] = entry.getValue().getSerialized();
+    for (MetadataEntry entry : store.values()) {
```

Comment:
Things like this are helpful, but in the future it is useful to do them separately. Very frequently random changes like this have been a point of contention and slowing down an otherwise-fine PR. They have also been a source of merge conflicts, so having them in a "cleanup PR" or a commit to itself makes them faster to review and handling the merge conflict easier.